### PR TITLE
Fix a bug in get_pairwise_compatibility_for_package returned self …

### PR DESCRIPTION
**Pull request by brianquinlan**
…compatibility.

### CODE REVIEW

File: compatibility_lib/compatibility_lib/fake_compatibility_store.py [hunk 1]
```diff
@@ -115,10 +115,15 @@ def get_pairwise_compatibility_for_package(self, package_name: str) -> \
...
             }.
         """
-        package_pairs = [[package.Package(package_name), package.Package(name)]
-                         for name in configs.PKG_LIST]
-        results = {frozenset(pair): self.get_pair_compatibility(pair)
-                   for pair in package_pairs}
+        package_pairs = [frozenset([package.Package(package_name), package.Package(name)])
+                         for name in configs.PKG_LIST
+                         if package_name != name]
+        print('package_pairs:', package_pairs)
```
Comment (ylil93) [inline compatibility_lib/compatibility_lib/fake_compatibility_store.py]: remove print statement

File: compatibility_lib/compatibility_lib/fake_compatibility_store.py [hunk 2]
```diff
@@ -115,10 +115,15 @@ def get_pairwise_compatibility_for_package(self, package_name: str) -> \
...
+        results = {pair: self.get_pair_compatibility(pair)
+                   for pair in package_pairs
+                   if self.get_pair_compatibility(pair)}
+        from pprint import pprint
+        pprint(results)
```
Comment (ylil93) [inline compatibility_lib/compatibility_lib/fake_compatibility_store.py]: remove print statement

File: compatibility_lib/compatibility_lib/fake_compatibility_store.py (hunk 1 above)
Comment (brianquinlan) [reply compatibility_lib/compatibility_lib/fake_compatibility_store.py]: I wasn't ready for review but fixed the prints and lint ;-)

File: compatibility_lib/compatibility_lib/fake_compatibility_store.py (hunk 2 above)
Comment (brianquinlan) [reply compatibility_lib/compatibility_lib/fake_compatibility_store.py]: Done

**Pull request merged**